manifests/platforms/eks.jsonnet

```diff
@@ -0,0 +1,147 @@
+/*
+ * Bitnami Kubernetes Production Runtime - A collection of services that makes it
+ * easy to run production workloads in Kubernetes.
+ *
+ * Copyright 2019 Bitnami
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Top-level file for AWS EKS
+
+local kube = import '../lib/kube.libsonnet';
```

Comment:
Use double-quotes (as existing entries), also fix several other lines below

Reply:
Funny, this was @anguslees undoing my changes (well, actually the changes by Visual Studio Code). Willdo.

Reply:
Whatever makes sense, as far as it's consistent.
Kinda obvious canonical plug:
https://dev.to/__shadz_/tabs-vs-space-49l5

Reply:
Still seeing those 2 style observations (single+double quotes mix[1], missing ending comma),
@falfaro: I'll wait for your fixes.

Reply:
Fixed.